README.rst

```diff
@@ -90,7 +90,64 @@ By default, blockstore is run as an app inside of Open edX. Enable it using the
+
+#. Prerequisite: Have an Open edX `Devstack <https://github.com/openedx/devstack>`_ properly installed and working. Your devstack must use the Nutmeg release of Open edX (or newer) or be tracking the ``master`` branch of ``edx-platform``.
+
+#. To run blockstore as an app inside of Open edX, enable it using the waffle switch `blockstore.use_blockstore_app_api <https://edx.readthedocs.io/projects/edx-platform-technical/en/latest/featuretoggles.html#featuretoggle-blockstore.use_blockstore_app_api>`_.
+
+#. Clone this repo in the ``src/`` directory, a sibling directory of your devstack folder.
+
+#. In your devstack directory, run the following commands in order:
+
+   .. code::
+      
+      # bring up the lms +studio containers
+      make dev.up.large-and-slow
+      # Bring up the studio shell
+      make dev.shell.studio
+      # Unistall the pypi blockstore
+      pip uninstall openedx-blockstore #You will need to confirm the unistall with "Y".
+      #Install your local blockstore.
+      pip install -e /edx/src/blockstore/  
+      exit
+      
+      # do the same in the LMS
+      make dev.shell.lms
+      # Unistall the pypi blockstore
+      pip uninstall openedx-blockstore #You will need to confirm the unistall with "Y".
+      #Install your local blockstore.
+      pip install -e /edx/src/blockstore/  
+      exit
+    
+      #restart the containers.
+      make dev.restart-devserver.lms dev.restart-devserver.studio
+
+      # you can see that your env now installed blockstore from a local directory by running:
+      make dev.shell.lms
+      pip list
+       
+      
+Seeing your changes will sometimes require running make requirements and then restarting the container.
+
+You're probably also going to want to create a collection for using content libraries, using:
+
+   .. code::
+   
+       # Create a "Collection" that new content libraries / xblocks can be created within:
+        docker exec -t edx.devstack.blockstore bash -c "source ~/.bashrc && echo \"from blockstore.apps.bundles.models import Collection; coll, _ = Collection.objects.get_or_create(title='Devstack Content Collection', uuid='11111111-2111-4111-8111-111111111111')\" | ./manage.py shell"
+       # Create an "Organization":
+       docker exec -t edx.devstack.lms bash -c "source /edx/app/edxapp/edxapp_env && echo \"from organizations.models import Organization; Organization.objects.get_or_create(short_name='DeveloperInc', defaults={'name': 'DeveloperInc', 'active': True})\" | python /edx/app/edxapp/edx-platform/manage.py lms shell"
+       
+The Library Authoring MFE will want to know about that collection id `using a ENV variable. <https://github.com/openedx/frontend-app-library-authoring/blob/d590aa2eb54c94b39d94f2ba12a6b458082c2e5e/.env#L19>`_
+       
+Then restart Studio and the LMS (``make dev.restart-devserver.lms dev.restart-devserver.studio``).
+
```

Comment:
The testing part of the readme is still a bit thin. What steps to follow and what to look for to confirm that a) the IDA is no longer running; b) the Django app is; and c) the Django app blockstore is running as expected.

Reply:
This guide doesn't care that you ever ran blockstore as a separate service. You could be running the blockstore service at the same time, but the waffle flag makes it such that it doesn't matter. This guide replaces that setup for local dev. 
I added a part to direct the user to set up content libraries v2. That is how blockstore is used and the system's effectiveness can be tested.